main.js

```diff
@@ -0,0 +1,78 @@
+/* eslint-disable no-use-before-define */
+// eslint-disable-next-line max-classes-per-file
+const listOfBooks = document.querySelector('.book-list');
+
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+    this.bookid = Math.random().toFixed(1);
```

Comment:
Because of this line of code, when adding new books, it'll be possible to have multiple books with the same id, and so, when removing one of them, in the local storage all of the items with the same id will be removed, but this will not reflect in the page unless it's refreshed. Kindly fix this issue.
- As you can see in the following screenshot, there are two books that have the id of 0.1: 
![Screenshot from 2023-03-20 19-46-19](https://user-images.githubusercontent.com/93520178/226438250-8d564d54-7148-4b3e-9b86-8f5a93404921.png)

Reply:
Noted, thank you for the ensightful suggestion, code modified.